src/server/tran_server.hpp

```diff
@@ -122,8 +122,9 @@ class tran_server
 	virtual log_lsa get_saved_lsa () const = 0; // used in active_tran_server
 
       protected:
-	connection_handler (tran_server &ts)
+	connection_handler (tran_server &ts, cubcomm::node &&node)
 	  : m_ts { ts }
+	  , m_node { std::move (node) }
```

Comment:
Nit.
If a `cubcomm::node` is mostly moved around and not stored in a vector anymore, I suppose some of its copy ctor/operators can be deleted.

Reply:
Actually, the whole `cubcomm::node` class is a bit superfluous now.

Reply:
I'v deleted copy things in this PR.